.github/workflows/test.yml

```diff
@@ -69,6 +69,10 @@ jobs:
       if: matrix.rootless != 'rootless'
       run: sudo -E PATH="$PATH" -- make localunittest
 
+    - name: unit test (i386)
```

Comment:
I think we should clarify the purpose of this in the YAML:
```
# We need to continue support for 32-bit ARM. However, we do not have 32-bit ARM CI, so we use i386 for testing 32bit stuff.
# We are not interested in providing official support for i386.
```

Reply:
Unfortunately everything that is nsenter / runc init related is not working. I tried to debug this but got nowhere so I'm putting this on hold for now. :(

Yet your point is valid; updated.